docs/content/guides/dagster/enriching-with-software-defined-assets.mdx

```diff
@@ -0,0 +1,407 @@
+---
+title: Upgrading To Software-Defined Assets | Dagster
+description: Understand when, why, and how to use software-defined assets (SDAs) in Dagster, as well as how SDAs work with other core Dagster concepts.
+---
+
+# Upgrading To Software-Defined Assets
+
+If you want to understand when, why, and how to use software-defined assets in Dagster, this guide is for you. We'll also demonstrate what some common Dagster jobs look like before and after using software-defined assets.
+
+Before we jump in, here's a quick refresher:
+
+- An **asset** is a persistent object in storage, such as a table, machine learning (ML) model, or file.
+- An [**op**](/concepts/ops-jobs-graphs/ops) is the core unit of computation in Dagster. For example, an op might accept tabular data as input and produce transformed tabular data as output.
+- A [**graph**](/concepts/ops-jobs-graphs/jobs-graphs) is a directed acyclic graph of ops or other graphs, which execute in order and pass data to each other.
+- A [**software-defined asset**](/concepts/assets/software-defined-assets) is a declaration of an asset that should exist and a description of how to compute it: the op or graph that needs to run and the upstream assets that it should run on.
+
+**Software-defined assets aren't a replacement for Dagster's core computational concepts**. Think of them as a top layer that links ops, graphs, and jobs to the long-lived objects they interact with.
```

Comment:
I think it would be helpful here to reiterate that ops are the core unit of computation that occurs within an asset.